src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php

```diff
@@ -77,6 +77,18 @@ protected function configure()
     /**
      * {@inheritdoc}
      */
+    public function isEnabled()
+    {
+        if (!class_exists('Symfony\Component\Asset\Package')) {
+            return false;
+        }
```

Comment:
I don't think this this command uses anything from the asset component and so could stay enabled if the component isn't installed.